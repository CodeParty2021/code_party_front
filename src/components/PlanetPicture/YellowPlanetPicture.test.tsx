import React from "react";
import { shallow } from "enzyme";
import { YellowPlanetPicture } from "./YellowPlanetPicture";
describe("<YellowPlanetPicture />", () => {
  it("YellowPlanetPicture snapshot test", () => {
    const wrapper = shallow(<YellowPlanetPicture size="160px" />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
