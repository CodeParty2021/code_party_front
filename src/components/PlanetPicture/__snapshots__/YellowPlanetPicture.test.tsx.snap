// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<YellowPlanetPicture /> YellowPlanetPicture snapshot test 1`] = `
Array [
  <svg
    fill="none"
    height="160px"
    viewBox="0 0 161 161"
    width="160px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      fill="#E5E5E5"
      height="161"
      width="161"
    />
    <path
      d="M48.2617 128.562C74.9156 146.257 110.867 138.994 128.562 112.34C146.256 85.6857 138.993 49.7341 112.339 32.0396C85.6854 14.345 49.7338 21.608 32.0393 48.262C14.3447 74.9159 21.6077 110.867 48.2617 128.562Z"
      fill="url(#paint0_linear_0_1)"
    />
    <path
      d="M84.6877 74.9974C84.1796 74.0127 83.974 72.8997 84.0969 71.7984C84.2197 70.6971 84.6654 69.6568 85.378 68.8082C86.0905 67.9596 87.038 67.3405 88.1015 67.0291C89.1649 66.7176 90.2966 66.7275 91.3544 67.0576C92.4122 67.3876 93.3488 68.0231 94.0464 68.884C94.744 69.745 95.1714 70.793 95.2749 71.8963C95.3784 72.9995 95.1534 74.1088 94.6282 75.0845C94.1029 76.0602 93.3009 76.8587 92.323 77.3798C91.6663 77.7297 90.9466 77.9457 90.2057 78.0152C89.4649 78.0847 88.7176 78.0064 88.0073 77.7847C87.297 77.5631 86.6378 77.2026 86.0679 76.7241C85.4981 76.2456 85.0289 75.6587 84.6877 74.9974V74.9974Z"
      fill="#FFC047"
    />
    <path
      d="M73.8268 137.868L29.2598 52.8914C24.7813 61.2338 22.4157 70.5467 22.37 80.0151C22.3243 89.4835 24.5999 98.8189 28.9976 107.204C33.3954 115.589 39.7812 122.769 47.5963 128.115C55.4115 133.461 64.4177 136.809 73.8268 137.868Z"
      fill="#FFC047"
    />
    <path
      d="M72.6243 94.3026C73.4491 93.8701 74.1805 93.2793 74.7769 92.564C75.3732 91.8487 75.8228 91.0229 76.0999 90.1338C76.377 89.2447 76.4762 88.3098 76.3918 87.3823C76.3074 86.4549 76.0411 85.5532 75.6081 84.7287L48.0483 32.1732C43.9948 34.8833 40.3017 38.0968 37.0573 41.7367L63.0478 91.3058C63.9209 92.9726 65.42 94.2246 67.2158 94.7865C69.0115 95.3485 70.9569 95.1744 72.6243 94.3026V94.3026Z"
      fill="#FFC047"
    />
    <path
      d="M109.888 77.4183C112.421 77.6295 114.86 78.4767 116.979 79.8813C119.098 81.2859 120.828 83.2026 122.009 85.4537C123.19 87.7049 123.784 90.2177 123.735 92.7593C123.686 95.3009 122.997 97.7891 121.731 99.9935C121.08 101.09 120.717 102.333 120.674 103.608C120.632 104.882 120.912 106.147 121.488 107.284L126.014 115.909C132.848 107.126 137.01 96.5607 138 85.4761C138.991 74.3915 136.77 63.256 131.601 53.4001C126.433 43.5443 118.536 35.3849 108.854 29.8969C99.1726 24.409 88.1157 21.8246 77.0046 22.4526L103.759 73.4616C104.362 74.586 105.241 75.5388 106.313 76.2308C107.385 76.9228 108.615 77.3314 109.888 77.4183V77.4183Z"
      fill="#FFC047"
    />
    <path
      d="M117.196 87.8786C116.328 86.2234 114.988 84.8629 113.346 83.9692C111.704 83.0755 109.835 82.6889 107.973 82.8582C106.112 83.0274 104.342 83.7449 102.889 84.92C101.435 86.095 100.363 87.6749 99.807 89.4595C99.2515 91.2442 99.2377 93.1536 99.7675 94.9461C100.297 96.7386 101.347 98.3337 102.783 99.5296C104.22 100.725 105.979 101.468 107.838 101.664C109.697 101.86 111.572 101.501 113.226 100.631C114.058 100.177 114.819 99.6023 115.482 98.9253C116.906 97.5058 117.84 95.6692 118.148 93.6826C118.456 91.6959 118.123 89.6627 117.196 87.8786V87.8786Z"
      fill="#FFC047"
    />
    <path
      d="M87.9938 77.7849C90.9758 78.7153 94.1475 77.0522 95.0779 74.0703C96.0083 71.0883 94.3452 67.9166 91.3632 66.9862C88.3812 66.0558 85.2096 67.7189 84.2792 70.7009C83.3488 73.6829 85.0119 76.8545 87.9938 77.7849Z"
      fill="#FFC047"
    />
    <defs>
      <linearGradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear_0_1"
        x1="136.417"
        x2="25.4745"
        y1="102.697"
        y2="68.3778"
      >
        <stop
          stopColor="#FFC555"
        />
        <stop
          offset="1"
          stopColor="#FDFF9D"
        />
      </linearGradient>
    </defs>
  </svg>,
]
`;
